public/apps/configuration/utils/audit-logging-view-utils.tsx

```diff
@@ -0,0 +1,211 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+import {
+  EuiDescribedFormGroup,
+  EuiFlexGrid,
+  EuiForm,
+  EuiFormRow,
+  EuiPanel,
+  EuiSpacer,
+  EuiSwitch,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import React from 'react';
+import { displayArray, displayBoolean, displayObject, renderTextFlexItem } from './display-utils';
+
+export function renderStatusPanel(onSwitchChange: any, auditLoggingEnabled: boolean) {
+  return (
+    <EuiForm>
+      <EuiDescribedFormGroup
+        title={<h3>Enable audit logging</h3>}
+        description={<>Enable or disable audit logging</>}
+      >
+        <EuiFormRow>
+          <EuiSwitch
+            name="auditLoggingEnabledSwitch"
+            label={auditLoggingEnabled ? 'Enabled' : 'Disabled'}
+            checked={auditLoggingEnabled}
+            onChange={onSwitchChange}
+          />
+        </EuiFormRow>
+      </EuiDescribedFormGroup>
+    </EuiForm>
+  );
+}
+
+export function renderGeneralSettings(generalConfiguration: any) {
+  return (
+    <>
+      {renderGeneralSettingsForLayer(generalConfiguration)}
+
+      <EuiSpacer />
+
+      {renderGeneralSettingsForAttribute(generalConfiguration)}
+
+      <EuiSpacer />
+
+      {renderGeneralSettingsForExclude(generalConfiguration)}
+
+      <EuiSpacer />
+
+      {renderGeneralSettingsForReport(generalConfiguration)}
+    </>
+  );
+}
+
+function renderGeneralSettingsForLayer(generalConfiguration: any) {
+  return (
+    <>
+      <EuiText>
+        <h3>Layer settings</h3>
+      </EuiText>
+
+      <EuiSpacer />
+
+      <EuiFlexGrid columns={2}>
+        {renderTextFlexItem(
+          'REST layer',
+          generalConfiguration.enable_rest ? 'Enabled' : 'Disabled'
+        )}
+        {renderTextFlexItem(
+          'Transport layer',
+          generalConfiguration.enable_transport ? 'Enabled' : 'Disabled'
+        )}
+        {renderTextFlexItem(
+          'REST disabled categories',
+          generalConfiguration.disabled_rest_categories?.join(', ')
+        )}
+        {renderTextFlexItem(
+          'Transport disabled categories',
+          generalConfiguration.disabled_transport_categories?.join(', ')
+        )}
```

Comment:
Why not use `displayBoolean` and `displayArray` here?

Reply:
Missed here. Added